diff -ur ./scip-3.0.2/make/make.install ../scipoptsuite-3.0.2-patched/scip-3.0.2/make/make.install
--- ./scip-3.0.2/make/make.install	2013-10-08 13:45:28.487934225 +0100
+++ ../scipoptsuite-3.0.2-patched/scip-3.0.2/make/make.install	2014-04-15 11:49:33.998548126 +0100
@@ -26,15 +26,22 @@
 installheaders:	
 ifneq ($(INSTALLDIR),)
 		@echo "-> install SCIP headers into INSTALLDIR <$(INSTALLDIR)>"
-		@-install -m 755 -d $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(INSTALLDIR)/$(INCLUDEDIR)/objscip/
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SCIPPLUGININCSRC) 
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SRCDIR)/scip/pub_*h 
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SRCDIR)/scip/struct_*h 
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SRCDIR)/scip/type_*h 
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SRCDIR)/scip/scip.h
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/objscip/ $(OBJSCIPINCSRC) 
+		@-install -m 755 -d $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(INSTALLDIR)/$(INCLUDEDIR)/objscip/ 
+		@-install -m 755 -d $(INSTALLDIR)/$(INCLUDEDIR)/blockmemshell/
+		@-install -m 755 -d $(INSTALLDIR)/$(INCLUDEDIR)/nlpi/
+		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SCIPPLUGININCSRC)
+		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SRCDIR)/scip/*.h 
+		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/objscip/ $(OBJSCIPINCSRC)
+		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/objscip/ $(SRCDIR)/objscip/*.h 
+		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/blockmemshell/ $(SRCDIR)/blockmemshell/*.h 
+		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/nlpi/ $(SRCDIR)/nlpi/*.h 
 endif
 
+SOPLEXLIBFILE=$(LIBDIR)/libsoplex.$(OSTYPE).$(ARCH).$(COMP).$(IPOPTOPT).$(LIBEXT)
+SOPLEXLIBFILE_WITHDUMMY=$(LIBDIR)/libsoplex-dummy.$(OSTYPE).$(ARCH).$(COMP).$(IPOPTOPT).$(LIBEXT)
+
+ZIMPLLIBFILE=$(LIBDIR)/libzimpl.$(OSTYPE).$(ARCH).$(COMP).$(IPOPTOPT).$(LIBEXT)
+ZIMPLLIBFILE_WITHDUMMY=$(LIBDIR)/libzimpl-dummy.$(OSTYPE).$(ARCH).$(COMP).$(IPOPTOPT).$(LIBEXT)
 
 
 .PHONY: installlibs
@@ -42,7 +49,9 @@
 ifneq ($(INSTALLDIR),)
 		@echo "-> install SCIP libraries into INSTALLDIR <$(INSTALLDIR)>"
 		@-install -m 755 -d $(INSTALLDIR)/$(LIBDIR)/
-		@-install -m 644 -p -t $(INSTALLDIR)/$(LIBDIR)/ $(LPILIBFILE) $(NLPILIBFILE) $(SCIPLIBFILE) $(OBJSCIPLIBFILE) 
+		@cp $(SOPLEXLIBFILE) $(SOPLEXLIBFILE_WITHDUMMY)
+		@cp $(ZIMPLLIBFILE) $(ZIMPLLIBFILE_WITHDUMMY)
+		@-install -m 644 -p -t $(INSTALLDIR)/$(LIBDIR)/ $(SOPLEXLIBFILE_WITHDUMMY) $(ZIMPLLIBFILE_WITHDUMMY) $(LPILIBFILE) $(NLPILIBFILE) $(SCIPLIBFILE) $(OBJSCIPLIBFILE) 
 endif
 
 
