diff -ur ./scip-3.0.2/make/make.install ../scipoptsuite/BUILD/scipoptsuite/src/scipoptsuite/scip-3.0.2/make/make.install
--- ./scip-3.0.2/make/make.install	2013-10-08 13:45:28.000000000 +0100
+++ ../scipoptsuite/BUILD/scipoptsuite/src/scipoptsuite/scip-3.0.2/make/make.install	2014-04-13 12:47:42.000000000 +0100
@@ -22,27 +22,40 @@
 install:	installheaders installlibs installbinary
 
 
+
+
 .PHONY: installheaders
 installheaders:	
 ifneq ($(INSTALLDIR),)
 		@echo "-> install SCIP headers into INSTALLDIR <$(INSTALLDIR)>"
-		@-install -m 755 -d $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(INSTALLDIR)/$(INCLUDEDIR)/objscip/
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SCIPPLUGININCSRC) 
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SRCDIR)/scip/pub_*h 
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SRCDIR)/scip/struct_*h 
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SRCDIR)/scip/type_*h 
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(SRCDIR)/scip/scip.h
-		@-install -m 644 -p -t $(INSTALLDIR)/$(INCLUDEDIR)/objscip/ $(OBJSCIPINCSRC) 
+		@-install -m 755 -d $(INSTALLDIR)/$(INCLUDEDIR)/scip/ $(INSTALLDIR)/$(INCLUDEDIR)/objscip/ 
+		@-install -m 755 -d $(INSTALLDIR)/$(INCLUDEDIR)/blockmemshell/
+		@-install -m 755 -d $(INSTALLDIR)/$(INCLUDEDIR)/nlpi/
+		@-install -m 644 -p $(SCIPPLUGININCSRC) $(INSTALLDIR)/$(INCLUDEDIR)/scip/
+		@-install -m 644 -p $(SRCDIR)/scip/*.h $(INSTALLDIR)/$(INCLUDEDIR)/scip/ 
+		@-install -m 644 -p $(OBJSCIPINCSRC) $(INSTALLDIR)/$(INCLUDEDIR)/objscip/ 
+		@-install -m 644 -p $(SRCDIR)/objscip/*.h $(INSTALLDIR)/$(INCLUDEDIR)/objscip/
+		@-install -m 644 -p $(SRCDIR)/blockmemshell/*.h $(INSTALLDIR)/$(INCLUDEDIR)/blockmemshell/
+		@-install -m 644 -p $(SRCDIR)/nlpi/*.h $(INSTALLDIR)/$(INCLUDEDIR)/nlpi/
 endif
 
 
 
+SOPLEXLIBFILE=$(LIBDIR)/libsoplex.$(OSTYPE).$(ARCH).$(COMP).$(IPOPTOPT).$(STATICLIBEXT)
+SOPLEXLIBFILE_WITHDUMMY=$(LIBDIR)/libsoplex-dummy.$(OSTYPE).$(ARCH).$(COMP).$(IPOPTOPT).$(STATICLIBEXT)
+
+ZIMPLLIBFILE=$(LIBDIR)/libzimpl.$(OSTYPE).$(ARCH).$(COMP).$(IPOPTOPT).$(STATICLIBEXT)
+ZIMPLLIBFILE_WITHDUMMY=$(LIBDIR)/libzimpl-dummy.$(OSTYPE).$(ARCH).$(COMP).$(IPOPTOPT).$(STATICLIBEXT)
+
+
 .PHONY: installlibs
 installlibs:	libs
 ifneq ($(INSTALLDIR),)
 		@echo "-> install SCIP libraries into INSTALLDIR <$(INSTALLDIR)>"
 		@-install -m 755 -d $(INSTALLDIR)/$(LIBDIR)/
-		@-install -m 644 -p -t $(INSTALLDIR)/$(LIBDIR)/ $(LPILIBFILE) $(NLPILIBFILE) $(SCIPLIBFILE) $(OBJSCIPLIBFILE) 
+		@cp $(SOPLEXLIBFILE) $(SOPLEXLIBFILE_WITHDUMMY)
+		@cp $(ZIMPLLIBFILE) $(ZIMPLLIBFILE_WITHDUMMY)
+		@-install -m 644 -p $(SOPLEXLIBFILE_WITHDUMMY) $(ZIMPLLIBFILE_WITHDUMMY) $(LPILIBFILE) $(NLPILIBFILE) $(SCIPLIBFILE) $(OBJSCIPLIBFILE) $(INSTALLDIR)/$(LIBDIR)/
 endif
 
 
@@ -52,7 +65,7 @@
 ifneq ($(INSTALLDIR),)
 		@echo "-> install SCIP binary into INSTALLDIR <$(INSTALLDIR)>"
 		@-install -m 755 -d $(INSTALLDIR)/$(BINDIR)/
-		@-install -m 755 -p -t $(INSTALLDIR)/$(BINDIR)/ $(MAINFILE) $(MAINLINK) $(MAINSHORTLINK)
+		@-install -m 755 -p $(MAINFILE) $(MAINLINK) $(MAINSHORTLINK) $(INSTALLDIR)/$(BINDIR)/
 endif
 
 
diff -ur ./scip-3.0.2/make/make.project ../scipoptsuite/BUILD/scipoptsuite/src/scipoptsuite/scip-3.0.2/make/make.project
--- ./scip-3.0.2/make/make.project	2013-10-08 13:45:49.000000000 +0100
+++ ../scipoptsuite/BUILD/scipoptsuite/src/scipoptsuite/scip-3.0.2/make/make.project	2014-04-12 21:40:05.000000000 +0100
@@ -114,7 +114,7 @@
 FLAGS		=	-I$(SRCDIR) -DWITH_SCIPDEF -I$(SCIPDIR)/src
 OFLAGS		=
 CFLAGS		=	
-CXXFLAGS	=	
+CXXFLAGS	=	-std=c++0x 
 LDFLAGS		=	$(LINKCC_l)m$(LINKLIBSUFFIX)
 ARFLAGS		=	cr
 DFLAGS		=	-MM
diff -ur ./soplex-1.7.2/make/make.darwin.gnu.opt ../scipoptsuite/BUILD/scipoptsuite/src/scipoptsuite/soplex-1.7.2/make/make.darwin.gnu.opt
--- ./soplex-1.7.2/make/make.darwin.gnu.opt	2013-10-16 15:02:17.000000000 +0100
+++ ../scipoptsuite/BUILD/scipoptsuite/src/scipoptsuite/soplex-1.7.2/make/make.darwin.gnu.opt	2014-04-12 21:40:05.000000000 +0100
@@ -1,2 +1,2 @@
 CPPFLAGS       +=	-DNDEBUG
-CXXFLAGS       +=      -O3 -ffast-math -fforce-addr -mtune=native -fexceptions -funroll-loops -fomit-frame-pointer #-march=
+CXXFLAGS       +=      -O3 -ffast-math -mtune=native -fexceptions -funroll-loops -fomit-frame-pointer #-march=
diff -ur ./zimpl-3.3.1/make/make.darwin.x86_64.gnu.opt ../scipoptsuite/BUILD/scipoptsuite/src/scipoptsuite/zimpl-3.3.1/make/make.darwin.x86_64.gnu.opt
--- ./zimpl-3.3.1/make/make.darwin.x86_64.gnu.opt	2011-10-31 08:48:56.000000000 +0000
+++ ../scipoptsuite/BUILD/scipoptsuite/src/scipoptsuite/zimpl-3.3.1/make/make.darwin.x86_64.gnu.opt	2014-04-12 21:40:05.000000000 +0100
@@ -1,6 +1,6 @@
 #--- $Id: make.darwin.x86_64.gnu.opt,v 1.6 2011/10/31 08:48:56 bzfkocht Exp $
 CPPFLAGS        +=      -DNDEBUG -DNO_MSHELL -D__NO_MATH_INLINES
-CFLAGS		+=	-O6 -m64 
+CFLAGS		+=	-O3 -m64 
 LDFLAGS         =       -lgmp -lz -lm 
 
 ifeq ($(LINK),shared)
